gtk2/gtk2.json

```
@@ -67,6 +67,10 @@
         {
           "type": "patch",
           "path": "gtk2-fix-crash-in-show-uri.patch"
+        },
+        {
+          "type": "patch",
+          "path": "gtk2-gcc14.patch"
```

nit
```suggestion
          "paths": [
              "gtk2-fix-crash-in-show-uri.patch",
              "gtk2-gcc14.patch"
          ]
```

actually merged with the previous one too (line 65) but github doesn't allow me to suggest it.